include/riak_kv_ts.hrl

```diff
@@ -59,7 +59,14 @@
 
 -record(riak_sql_describe_v1,
         {
-          'DESCRIBE' = <<>>  :: binary()
+          'DESCRIBE'    = <<>>  :: binary()
+        }).
+
+-record(riak_sql_insert_v1,
+        {
+          'INSERT'      = <<>>  :: binary(),
+          fields                :: [{identifier, binary()}],
+          values                :: list(list(term()))
```

Comment:
There's a type for that: `riak_pb_ts_codec:ldbvalue()`, which would be appropriate for the inner `term()`.